### PR TITLE
feat(cph): supports parameters related to cph server 2.0 flavor

### PR DESCRIPTION
**Pull request by luoping-12345**
**What this PR does / why we need it**:
 supports parameters related to cph server 2.0 flavor

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```
 supports parameters related to cph server 2.0 flavor
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST="./huaweicloud/services/acceptance/cph" TESTARGS="-run TestAccCphServer_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cph -v -run TestAccCphServer_basic -timeout 360m -parallel 4
=== RUN   TestAccCphServer_basic
=== PAUSE TestAccCphServer_basic
=== CONT  TestAccCphServer_basic
--- PASS: TestAccCphServer_basic (2075.59s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cph       2075.646s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: huaweicloud/services/cph/resource_huaweicloud_cph_server.go [hunk 1]
```diff
@@ -164,6 +164,20 @@ func ResourceCphServer() *schema.Resource {
 				ForceNew:    true,
 				Description: `The application port enabled by the cloud phone.`,
 			},
+			"phone_data_volume": {
+				Type:        schema.TypeList,
+				Elem:        cphPhoneDataVolumeSchema(),
+				Optional:    true,
```
Comment (saf3dfsa) [inline huaweicloud/services/cph/resource_huaweicloud_cph_server.go:170]: remove the optional

File: docs/resources/cph_server.md [hunk 2]
```diff
@@ -161,6 +167,20 @@ The `ApplicationPort` block supports:
...
+
+* `volume_type` - (Optional, String) Specifies the volume type.
+
+<a name="server_share_data_volume"></a>
+The `server_share_data_volume` block supports:
+
+* `volume_type` - (Optional, String) Specifies the share volume type.
+
+* `size` - (Optional, Int) Specifies the share volume size, the unit is GB.
+
```
Comment (saf3dfsa) [inline docs/resources/cph_server.md]: They are only used in Read?

File: huaweicloud/services/acceptance/cph/resource_huaweicloud_cph_server_test.go [hunk 3]
```diff
@@ -188,6 +209,16 @@ resource "huaweicloud_cph_server" "test" {
...
   }
 
+  phone_data_volume {
+    volume_type = "GPSSD"
+    volume_size = 100
+  }
+
+  server_share_data_volume {
+    volume_type = "GPSSD"
+    size = 100
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/cph/resource_huaweicloud_cph_server_test.go]:
```suggestion
    volume_type = "GPSSD"
    size        = 100
```

File: huaweicloud/services/cph/resource_huaweicloud_cph_server.go [hunk 4]
```diff
@@ -164,6 +164,20 @@ func ResourceCphServer() *schema.Resource {
...
+				Computed:    true,
+				Description: `The phone data volume.`,
+			},
+			"server_share_data_volume": {
+				Type:        schema.TypeList,
+				Elem:        cphShareDataVolumeSchema(),
+				Optional:    true,
+				Computed:    true,
+				Description: `The server share data volume.`,
+			},
```
Comment (houpeng80) [inline huaweicloud/services/cph/resource_huaweicloud_cph_server.go]: add forcenew?

**Pull request merged**